docs/en/trust.rst

```diff
@@ -747,6 +724,227 @@ Each JWT containing a Trust Chain in the JWT headers can be verified over time,
+            Public Key Algorithm: rsaEncryption
+                Public-Key: (2048 bit)
+                Modulus:
+                    00:af:82:3b:...
+                Exponent: 65537 (0x10001)
+        X509v3 extensions:
+            X509v3 Basic Constraints: 
+                CA:TRUE, pathlen:1
+            X509v3 Key Usage: 
+                Digital Signature, Key Encipherment
+            X509v3 Subject Alternative Name: 
+                URI:https://leaf.example.com
+            X509v3 Name Constraints: 
+                Permitted:
+                  URI.1=https://leaf.example.com
+                  DNS.1=leaf.example.com
+                Excluded:
+                  DNS=localhost
+                  DNS=localhost.localdomain
+                  DNS=127.0.0.1
+                  DNS=example.com
+                  DNS=example.org
+                  DNS=example.net
+                  DNS=*.example.org
+            X509v3 CRL Distribution Points: 
+                Full Name:
+                  URI:https://trust-ancor.example.com/crl/leaf.example.org.crl
+
+    Signature Algorithm: sha256WithRSAEncryption
+         5c:4f:3b:...
+
+
+    Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number: 987654321 (0x3ade68b1)
+    Signature Algorithm: sha256WithRSAEncryption
+        Issuer: CN=https://leaf.example.org, O=Leaf, C=IT
+        Validity
+            Not Before: Sep 1 00:00:00 2023 GMT
+            Not After : Sep 1 00:00:00 2024 GMT
+        Subject: CN=https://leaf.example.org, O=Leaf, C=IT
+        Subject Public Key Info:
+            Public Key Algorithm: rsaEncryption
+                Public-Key: (2048 bit)
+                Modulus:
+                    00:af:82:3b:...
+                Exponent: 65537 (0x10001)
+        X509v3 extensions:
+            X509v3 Basic Constraints: 
+                CA:FALSE
+            X509v3 Key Usage: 
+                Digital Signature, Key Encipherment
+            X509v3 Subject Alternative Name: 
+                URI:https://leaf.example.org
+            X509v3 Name Constraints: 
+                Permitted:
+                  URI.1=https://leaf.example.com
+                  DNS.1=leaf.example.com
+            X509v3 CRL Distribution Points: 
+                Full Name:
+                  URI:https://leaf.example.org/crl/self.crl
+
+    Signature Algorithm: sha256WithRSAEncryption
+         7d:6e:5f:...
+
+
+Federation participants can ensure that their certificates are consistent, enabling interoperability and security across the federation. This approach, enabling X.509 certificate issuance delegation, introduces innovative practices for certificate management using the underlying trust relationships established within the OpenID Federation.
+
+
+X.509 Certificate Revocation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+An X.509 Certificate can be revoked by its Issuer.
+When the X.509 Certificate issuer is the Leaf and therefore the X.509 Certificate is about itself, it MUST update its CRL.
```

Comment:
```suggestion
Revocation lists, and or any other revocation check mechanisms, are required only for X.509 Certificate with expiration time superior to 24 hours, otherwise they are not required.

When the X.509 Certificate issuer is the Leaf and therefore the X.509 Certificate is about itself, if the certificate expiration time is superior than 24 hours from the ``X509_NOT_VALID_BEFORE`` time, it MUST implement a CRL about the issued certificate and keep it updated.
```